components/common/Text/Text.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { Story, Meta } from '@storybook/react/types-6-0';
+import TextBody from './TextBody';
+import { TextProps } from './TextProps';
+import TextBold from './TextBold';
+import TextButton from './TextButton';
+import TextH1 from './TextH1';
+import TextH2 from './TextH2';
+import TextH3 from './TextH3';
+import TextPreTitle from './TextPreTitle';
+import TextSmall from './TextSmall';
+import TextSubtitle from './TextSubtitle';
+
+export default {
+  title: 'Components/Text',
```

Comment:
It would be better off adding storybook color arg:
  `argTypes: {
    color: {
      control: 'color'
    }
  }`

Reply:
added storybook color arg